packages/node/src/handlers/test-api.ts

```diff
@@ -9,17 +9,27 @@ import { WORKER_CALL_API_TIMEOUT } from '../constants';
 
 export async function testApi(
   config: Config,
-  endpointName: string,
+  endpointId: string,
   parameters: Record<string, string>
 ): Promise<[Error, null] | [null, ApiCallResponse]> {
   const testCallId = randomString(8);
   const masterHDNode = wallet.getMasterHDNode();
 
   const logOptions = logger.buildBaseOptions(config, { requestId: testCallId });
 
-  const triggerRequest = find(config.triggers.request, ['endpointName', endpointName]);
+  const triggerRequest = find(config.triggers.request, ['endpointId', endpointId]);
   if (!triggerRequest) {
-    return [new Error(`No such endpoint '${endpointName}'`), null];
+    return [new Error(`No such endpoint with ID '${endpointId}'`), null];
+  }
+
+  const endpoints = find(config.ois, ['title', triggerRequest.oisTitle])?.endpoints;
+  const endpoint = find(endpoints, ['name', triggerRequest.endpointName]);
```

Comment:
What happens if this `endpoints` is undefined? Maybe we need handle that error too before the second `find()`?

Reply:
We don't actually. Lodash's `find` is ok with `undefined` and it returns `undefined`, same as if the endpoint can't be found.

Reply:
oh nice 👍🏻